content/blog/2022-03-31-training-and-saving-models-on-a-dedicated-runner.md

```diff
@@ -0,0 +1,380 @@
+---
+title: Training and saving models with CML on a dedicated runner
+date: 2022-03-31
+description: Hello World
+descriptionLong: |
+  Some long
+  multiline
+  text (supports _basic_ [Markdown](link))
+picture: 2015-05-01/post-image.jpeg
+pictureComment: Some _Comment_ (supports _basic_ [Markdown](link))
+author: rob_dewit
+# commentsUrl: https://discuss.dvc.org/t/february-22-community-gems/1078
+tags:
+  - CML
+  - DVC
+  - DVC Remotes
+  - Git
+  - Pipelines
+---
+
+# Training and saving models with CML on a dedicated runner
+
+When you first develop a machine learning model, you will probably do so on your
+local machine. You can easily change algorithms, parameters, and input data
+right in your text editor, notebook, or terminal. But what happens when you
+already have a model deployed and want to run multiple experiments? Or if you
+want to deploy a new version daily?
+
+[Continuous Machine Learning (CML)](https://cml.dev/) helps you with this. It is
+an open-source library for implementing continuous integration and delivery
+(CI/CD) in machine learning projects. This way we can define a pipeline to train
+a model and keep track of various versions.
+
+> Imagine we work for the fraud detection team of a webshop and we want to build
+> a model to detect fraudulent orders. One week fraudsters might be trying to
+> order gaming consoles, while the next they might go all-in on diapers. In that
+> case, we would probably want to retrain our model regularly to catch this
+> [drift](https://en.wikipedia.org/wiki/Concept_drift). And we would also want
+> to use the most up-to-date version of our model in production.
```

Comment:
We usually use this for a quote, but like that this is broken out like this.  @julieg18 @rogermparent How can we make a callout box, maybe in a color? (pastel branding colors, I found [this](https://rdmd.readme.io/docs/callouts#section--examples-), and tried out the examples, but not doing it right with the CSS).  And an emoji (like you did below, but needs to be different emoji than 💡 if we use that there.  Something signifying "setting"

Reply:
https://github.com/iterative/dvc.org/issues/3153